core/templates/domain/utilities/entity-context.model.ts

```diff
@@ -36,17 +33,8 @@ export class EntityContext {
   getType(): string {
     return this._type;
   }
-}
 
-@Injectable({
-  providedIn: 'root'
-})
-export class EntityContextObjectFactory {
-  create(id: string, type: string): EntityContext {
+  static create(id: string, type: string): EntityContext {
```

Comment:
Should we have this as a constructor for the class?

Reply:
Done